src/pages/verify/integrations/javascript.mdx

```diff
@@ -0,0 +1,292 @@
+---
+product: verify
+category: Integrations
+sort: 0
+title: JavaScript
+subtitle: Accept MitID, NemID, Swedish BankID, Norwegian BankID and more e-ID logins in your JavaScript application with @criipto-auth.js.
+---
+
+import Layout from '../../../layouts/mdx';
+export default Layout;
+
+This guide shows how to use Criipto Verify in a JavaScript application with plain HTML.
+
+You can also use `@criipto-auth.js` with Vue.js, Angular and other frontend frameworks, and we will be adding more integration guides in the future. For React applications, you can use our [@criipto/verify-react library](/verify/integrations/react) instead.
+
+The full documentation is available [on Github](https://github.com/criipto/criipto-auth.js).
+
+## Register Your Application in Criipto Verify
+
+After you signed up for Criipto Verify, you must [register an application](https://dashboard.criipto.com/applications/add) before you can try logging in with any e-ID.
+
+Once your application is registered, the dashboard will contain the necessary details to configure your application to communicate with Criipto Verify.
+You will need the following information from the General tab of your application settings:
+
+- _Client ID_ to identify your application to Criipto Verify.
+- _Domain_ on which you will be communicating with Criipto Verify.
+
+### Register callback URLs
+
+Before you can start sending authentication requests to Criipto Verify, you need to register the URLs on which you want to receive the returned *JSON Web Token*, JWT.
+
+The Callback URL of your application is the URL where Criipto Verify will redirect to after the user has authenticated in order for the OpenID Connect middleware to complete the authentication process.
+
+You will need to add this URL to the list of allowed URLs for your application.
+
+If you are developing with plain HTML and running your application locally, your callback URL might look like this:
+
+```
+http://localhost:3000/example/index.html
+```
+
+When you deploy your application to a different URL, you will also need to ensure you add that URL to the Callback URLs field in your application settings.
+
+## Install SDK
+
+You can include the library directly into your HTML file by loading it from [unpkg](https://www.npmjs.com/package/unpkg):
+
+```html
+<!-- index.html -->
+
+<script src="https://unpkg.com/@criipto/auth-js"></script>
```

Comment:
This resolves to a `.cjs`, it must resolve to the umd file when used in a browser

Reply:
Ah, I can add a `unpkg` tag in `package.json` - will do so.